lib/schema_plus/views/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -2,17 +2,12 @@ module SchemaPlus::Views
   module ActiveRecord
     module ConnectionAdapters
       module PostgresqlAdapter
-
-        def views(name = nil) #:nodoc:
-          SchemaMonkey::Middleware::Schema::Views.start(connection: self, query_name: name, views: []) { |env|
-            sql = <<-SQL
-            SELECT viewname
-              FROM pg_views
-            WHERE schemaname = ANY (current_schemas(false))
-            AND viewname NOT LIKE 'pg\_%'
-            SQL
-            sql += " AND schemaname != 'postgis'" if adapter_name == 'PostGIS'
```

Comment:
Here's the thread and PR that introduced it: SchemaPlus/schema_plus#94.  I guess it's analogous to suppressing the `pr_*` tables.  I'm not familiar with PostGIS either but over the years we've had occasional people raise issues/submit PRs to deal with it.   Possibly this too could be maintained via middleware if it's still an issue?